From: Mike Kravetz <mike.kravetz@oracle.com>
Subject: MAINTAINERS: remove hugetlb maintainer Mike Kravetz
Date: Wed, 20 Dec 2023 14:08:43 -0800

I am stepping away from my role as hugetlb maintainer.  There should be no
gap in coverage as Muchun Song is also a hugetlb maintainer.

Link: https://lkml.kernel.org/r/20231220220843.73586-1-mike.kravetz@oracle.com
Signed-off-by: Mike Kravetz <mike.kravetz@oracle.com>
Cc: Muchun Song <songmuchun@bytedance.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 MAINTAINERS |    1 -
 1 file changed, 1 deletion(-)

--- a/MAINTAINERS~maintainers-remove-hugetlb-maintainer-mike-kravetz
+++ a/MAINTAINERS
@@ -9760,7 +9760,6 @@ F:	Documentation/networking/device_drive
 F:	drivers/net/ethernet/huawei/hinic/
 
 HUGETLB SUBSYSTEM
-M:	Mike Kravetz <mike.kravetz@oracle.com>
 M:	Muchun Song <muchun.song@linux.dev>
 L:	linux-mm@kvack.org
 S:	Maintained
_
